code/API_definitions/click-to-dial.yaml

```
@@ -478,7 +470,7 @@ components:
           description: The type of the event.
           type: string
           enum:
-            - "org.camaraproject.qod-provisioning.v0.status-changed"
+            - "org.camaraproject.clicktodial.v0.status-changed"
```

The API name is `click-to-dial `(as defined in line 42)

```suggestion
            - "org.camaraproject.click-to-dial.v0.status-changed"
```